src/dstore/dstore_base.c

```diff
@@ -116,53 +116,99 @@ int dstore_obj_delete(struct dstore *dstore, void *ctx,
 	return dstore->dstore_ops->obj_delete(dstore, ctx, oid);
 }
 
-int dstore_obj_resize(struct dstore_obj *obj, size_t old_size, size_t new_size)
+/* TODO: Can be removed when plugin API for removing objects from backend store
+ * is implemented
+ */
+#define MAX_IO_SIZE (1024*1024)
```

Comment:
Shall we rename it to DSAL_MAX_IO_SIZE or DSTORE_MAX_IO_SIZE?
MAX_IO_SIZE is too generic.

Reply:
Already a TODO to remove this, if it is permanent than naming can be addressed and it is not specific to DSAL or DSTORE so I would like to keep as it is.

Reply:
As discussed offline,
```
problem with the name is open source contributors will use our code now and it's possible that their modules might  
have this generic name "MAX_IO_SIZE" and might cause conflicts and bugs
```
So have addressed this comment.